src/v/cluster/feature_manager.cc

```diff
@@ -544,6 +516,55 @@ feature_manager::update_license(security::license&& license) {
     co_return make_error_code(errc::success);
 }
 
+model::enterprise_feature_report
```

Comment:
i'd guess that this delta would be significantly smaller if first added the report generation `license_required_feature_enabled` and return `.any()` (without moving the code down further in the file, then renamed the function in place with the `license_required_feature_enabled` right above it.

Reply:
fair point!

Reply:
done